README.md

```diff
@@ -5,40 +5,43 @@ Pipelines to automatically import updates to base images and scan them for vulne
+| `kubeshop/testkube-api-server`         | `hmctspublic.azurecr.io/imported/kubeshop/testkube-api-server`         |
+| `kubeshop/testkube-dashboard`          | `hmctspublic.azurecr.io/imported/kubeshop/testkube-dashboard`          |
+| `kubeshop/testkube-operator`           | `hmctspublic.azurecr.io/imported/kubeshop/testkube-operator`           |
+| `linuxserver/openssh-server`           | `hmctspublic.azurecr.io/imported/linuxserver/openssh-server`           |
+| `mailhog/mailhog`                      | `hmctspublic.azurecr.io/imported/mailhog/mailhog`                      |
+| `minio/minio`                          | `hmctspublic.azurecr.io/imported/minio/minio`                          |
+| `nats`                                 | `hmctspublic.azurecr.io/imported/nats`                                 |
+| `natsio/nats-server-config-reloader`   | `hmctspublic.azurecr.io/imported/natsi/nats-server-config-reloader`    |
+| `natsio/prometheus-nats-exporter`      | `hmctspublic.azurecr.io/imported/natsio/prometheus-nats-exporter`      |
+| `neuvector/controller`                 | `hmctspublic.azurecr.io/imported/neuvector/controller`                 |
+| `neuvector/enforcer`                   | `hmctspublic.azurecr.io/imported/neuvector/enforcer`                   |
+| `neuvector/manager`                    | `hmctspublic.azurecr.io/imported/neuvector/manager`                    |
+| `neuvector/updater`                    | `hmctspublic.azurecr.io/imported/neuvector/updater`                    |
+| `netboxcommunity/netbox`               | `hmctspublic.azurecr.io/imported/netboxcommunity/netbox`               |
+| `nginx`                                | `hmctspublic.azurecr.io/imported/nginx`                                |
+| `node`                                 | `hmctspublic.azurecr.io/imported/library/node`                         |
 | `otel/opentelemetry-collector-contrib` | `hmctspublic.azurecr.io/imported/otel/opentelemetry-collector/contrib` |
-| `prom/node-exporter`                 | `hmctspublic.azurecr.io/imported/prom/node-exporter`       |
-| `traefik`                            | `hmctspublic.azurecr.io/imported/traefik`                 |
-| `willwill/kube-slack`                | `hmctspublic.azurecr.io/imported/willwill/kube-slack`     |
+| `prom/node-exporter`                   | `hmctspublic.azurecr.io/imported/prom/node-exporter`                   |
+| `traefik`                              | `hmctspublic.azurecr.io/imported/traefik`                              |
+| `willwill/kube-slack`                  | `hmctspublic.azurecr.io/imported/willwill/kube-slack`                  |
+| `postgres`                             | `hmctspublic.azurecr.io/imported/postgres`                             |
+| `toolbelt/oathtool`                    | `hmctspublic.azurecr.io/imported/toolbelt/oathtool`                    |
+| `jqlang/jq`                            | `hmctspublic.azurecr.io/imported/jqlang/jq`                            |
```

Comment:
Should these be in alphabetical order rather than just at the end of the list?  The existing entries in the table are in alphabetical order.

Also add in elasticsearch/elasticsearch, neuvector/scanner and testcontainers/ryuk.